corehq/motech/fhir/tasks.py

```diff
@@ -69,9 +69,10 @@ def run_monthly_importers():
             frequency=IMPORT_FREQUENCY_MONTHLY
         ).select_related('connection_settings').all()
     ):
-        run_importer(importer)
+        run_importer.delay(importer)
 
 
+@task(serializer='pickle', queue='background_queue', ignore_result=True)
```

Comment:
What's the reason for 🥒 here? We are trying not to add this to new tasks: https://commcare-hq.readthedocs.io/celery.html#best-practices

Reply:
Ooo! I see I violated both best practices! 275d264207c